### PR TITLE
Revert "Geo Aardvark: Reverse the boolean logic for gbl_georeferenced_b"

### PR DESCRIPTION
**Pull request by thatbudakguy**
This reverts commit 91e574a9e967d2b8af87252b51bedb86645b361b.

### CODE REVIEW

File: spec/integration/geo_aardvark_config_spec.rb [hunk 1]
```diff
@@ -296,18 +296,6 @@
...
   end
 
-  context 'with a scanned map that has a georeferenced map' do
-    let(:druid) { 'gm036df2527' }
-
-    it 'maps the georeferenced status' do
-      expect(result['gbl_georeferenced_b']).to eq true
-    end
-
-    it 'populates the link to the georeferenced map in dct_relation_sm' do
```
Comment (marlo-longley) [inline spec/integration/geo_aardvark_config_spec.rb:306]: Is dct_relations_sm modified in this PR? Might be missing something.

Comment (thatbudakguy) [reply spec/integration/geo_aardvark_config_spec.rb:306]: No, but this change was apparently part of the commit that got reverted, so it also was reverted...I'll see if I can fix that with another commit

Comment (marlo-longley) [reply spec/integration/geo_aardvark_config_spec.rb:306]: Well then it's my bad for a messy commit! We can add it later if easier with the git.

Comment (thatbudakguy) [reply spec/integration/geo_aardvark_config_spec.rb:306]: I was able to fix it and restore the fixture for this test also. Makes the PR a little cleaner to review.

**Review by marlo-longley — APPROVED** [review]

**Pull request merged**